### PR TITLE
adding falco release post

### PR DESCRIPTION
**Pull request by DanielMcSheehy**
[ch1800]
Screen Shot 2021-04-28 at 9 09 40 AM
Screen Shot 2021-04-28 at 9 09 49 AM

### CODE REVIEW

File: product-releases/2021-04-28-falco-support.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+author_title: Software Engineer
+author_url: https://github.com/DanielMcSheehy
+author_image_url: https://avatars.githubusercontent.com/u/28493438?s=460&u=4bbd1de69b9ca85d21e45350d15db419a27072fc&v=4
+tags: [security]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Last week, the effx platform engineering team landed support for [Falco](https://falco.org/).
+Falco is a popular open source security and complicance tool, and is listed as the
```
Comment (mjpitz) [inline product-releases/2021-04-28-falco-support.mdx:13]: This is a bit of a run-on sentence.

```
Falco is a popular open-source security and compliance tool.
In fact, it's listed as the [defacto Kubernetes threat detection engine](#link).
The Falco integration displays real-time vulnerabilities when microservices do not meet compliance.
```

File: product-releases/2021-04-28-falco-support.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Falco is a popular open source security and complicance tool, and is listed as the 
+de facto Kubernetes threat detection engine. 
+The Falco integration displays real time security threats, and when microservices are not meeting
+their security compliance. 
+
+[Falco]: https://falco.org/
+
+<!--truncate-->
+
+View security alerts, and audits for all your services on the event feed.
```
Comment (mjpitz) [inline product-releases/2021-04-28-falco-support.mdx:22]: and vulnerabilities for all your services.

File: product-releases/2021-04-28-falco-support.mdx (hunk 1 above)
Comment (DanielMcSheehy) [reply product-releases/2021-04-28-falco-support.mdx]: ah, that's much better

File: product-releases/2021-04-28-falco-support.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+author: Daniel McSheehy
+author_title: Software Engineer
+author_url: https://github.com/DanielMcSheehy
+author_image_url: https://avatars.githubusercontent.com/u/28493438?s=460&u=4bbd1de69b9ca85d21e45350d15db419a27072fc&v=4
+tags: [security]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Falco is a popular open-source security and compliance tool.
```
Comment (mjpitz) [inline product-releases/2021-04-28-falco-support.mdx:12]: why drop the first line? I like that announcement type of line.

Comment (DanielMcSheehy) [reply product-releases/2021-04-28-falco-support.mdx:14]: ah, misunderstood. added back

File: product-releases/2021-05-03-falco-support.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Set a security standard to make sure your services are always compliant.
+
+<p>
+  <img alt="feed screenshot"
+      src={useBaseUrl('img/integrations/falco-standard.png')}/>
+</p>
+
+Curious about how it works?
+[Sign up](http://app.effx.com/sign_up) today and give it a try!
```
Comment (Ttsunekawa) [inline product-releases/2021-05-03-falco-support.mdx:37]: nit, but since we have the links below, we don't have to put it in parentheses here

**Review by Ttsunekawa — APPROVED** [review]

**Review by mjpitz — APPROVED** [review]

**Pull request merged**